timeseries/src/test/groovy/com/hurence/logisland/timeseries/timeseries/SaxAnalyserTest.groovy

```diff
@@ -0,0 +1,30 @@
+package com.hurence.logisland.timeseries.timeseries
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import static com.hurence.logisland.timeseries.sax.SaxAnalyzer.anomalyDetect
+import static com.hurence.logisland.timeseries.sax.SaxAnalyzer.saxThreshold
+
+class SaxAnalyserTest extends Specification{
```

Comment:
In the test all existing functions should be tested with inputs and outputs ;) not forgetting empty inputs ;)

Reply:
In our case the other functions are used by one last function. So testing the last function is testing the functions used in it. So I need to create other tests for internal used function or not ?